### ISSUE TITLE
System freeze after a while

### ISSUE THREAD

**Issue by SpidFightFR** [Weather-OS/BRTXOnLinux#3]
Hey, i've tested your work for quite some time now...

I noticed that i get some kind of full system freezes after playing for a while, it happens randomly and the only solution i've got is to hard reset my pc...

Have you ever got the same issue on your system ? 

Also, is there some kind of way for me to check the patch you applied to vkd3d please ? I think such issue could be fixed with vkd3d latest version.

Have a nice day !

**Comment by Weather-OS** [comment]
Yes I have experienced this. This is due to an IRQ heap failure within RADV which results in a driver panic. 
There are some ways you can remedy this issue. First by entirely removing entities and preventing them from spawning, you can also use gamescope as it increases stability a bit.
Only the proprietary drivers are fully stable as of right now. We have to see if more patches land in RADV in order to see MCRTX become more stable.

The latest version of vkd3d includes the patch necessary for MCRTX and the one provided in this project is no longer required.

**Comment by SpidFightFR** [comment]
Well thanks for the intel, i'm going to try with gamescope, make some tests here and there... I'll keep you in touch!
